script/enable-gcloud.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
not needed either if we don't do the GKE build (remove all the scripts that are not necessary)